helm/templates/worker/deployment.yaml

```diff
@@ -32,6 +34,10 @@ spec:
           volumeMounts:
             - mountPath: /var/run/worker
               name: run-volume
+            - mountPath: /tmp
+              name: tmp-dir
```

Comment:
I just realized we've already a volume to be used by the worker data. Can we configure trivy to make use of this directory instead?